expertino/pddl/transport.jinja.pddl

```diff
@@ -0,0 +1,49 @@
+   ;; Transport a workpiece from one machine side to another
+   (:durative-action transport-step-1-drive-to
```

Comment:
maybe would be good to indicate somehow that these are sub actions that we don't want to use for planning but maybe something for later

Reply:
yea that thingy was copy-pasted from the main action, hence confusing. I fixed the comment.